app/utilities/Resources.java

```diff
@@ -0,0 +1,19 @@
+package utilities;
+
+/**
+ * Created by test on 02/02/16.
+ */
+public class Resources {
```

Comment:
This can also be marked as final to forbid inheritance and can have private constructor to forbid instantiation (utility class). :)